README.md

```diff
@@ -1,3 +1,39 @@
 # RHMAP Nagios for Docker
 
+[![Build Status](https://travis-ci.org/feedhenry/nagios-container.svg?branch=master)](https://travis-ci.org/feedhenry/nagios-container)
+
 Nagios server in a Docker container for RHMAP.
+
+## Development
+
+Install dependencies:
+```
+pip install pep8
+pip install autopep8
+```
+
+Run tests:
+
+```
+python -m unittest discover -s plugins/default
+````
+
+Style Guide:
+
+All python modules should conform to the PEP8 style guide https://www.python.org/dev/peps/pep-0008/
+
+To verify:
+
+```
+pep8 plugins/ --show-source
```

Comment:
@mikenairn  Ran through the commands in this guide, for the pep8 command, a bunch of E501 errors were displayed for me locally. I know these are ignored as part of the travis build itself, but could be confusing to anyone that is unaware of this. Maybe a note about E501 would be helpful